sdk/storage/blobs/samples/src/blobs_client_example.c

```diff
@@ -18,9 +18,26 @@
 #define URI_ENV "test_uri"
 
 int exit_code = 0;
+static az_span content_to_upload = AZ_SPAN_LITERAL_FROM_STR("Some test content");
+
+// Uncomment below code to enable logging (and the first lines of main function)
+/*
+static void test_log_func(az_log_classification classification, az_span message)
+{
+  (void)classification;
+  printf("%.*s\n", az_span_length(message), az_span_ptr(message));
```

Comment:
az_span_size, not length
```suggestion
  printf("%.*s\n", az_span_size(message), az_span_ptr(message));
```

Reply:
addressed in https://github.com/Azure/azure-sdk-for-c/pull/603